components/collaboration/collaboration-details/modal/BrandModal.tsx

```diff
@@ -0,0 +1,133 @@
+import { Image, Pressable, Linking } from "react-native";
+import { useTheme } from "@react-navigation/native";
+
+import { Text, View } from "@/components/theme/Themed";
+import stylesFn from "@/styles/modal/UploadModal.styles";
+import { Chip, Modal } from "react-native-paper";
+import Colors from "@/constants/Colors";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { faCheckCircle } from "@fortawesome/free-solid-svg-icons";
+
+interface BrandModalProps {
+  brandName: string;
+  brandDescription: string;
+  brandImage: string;
+  brandWebsite: string;
```

Comment:
Instead of passing props like this, we can create an object of related property which will be more readable -
Example - 
```
brand: {
 name: string;
 description: string;
 image: string;
 website: string;
 verified: boolean;
 category: string[];
}
```